### DISCUSSION TITLE
Git error on Application creation

### DISCUSSION THREAD

**Discussion by LefortGildas** [symfony/symfony#49281] [Q&A] [answered]
I'm using CLI downloaded today, I'm trying to create a new application with the following command : 
symfony new --webapp my_blog
And I get the following message : 
Git is a requirement and it cannot be found, please install it first.

GitHub Desktop Version 3.1.6 (x64) is running, do I need to install something else ?

**Comment by MatTheCat** [comment, marked as answer]
GitHub Desktop does not embed Git anymore. See https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/setting-up-github-desktop#part-3-configuring-git

**Reply by LefortGildas** [reply]
Thank you.
